pkg/apis/policies/v1alpha2/policy_types.go

```diff
@@ -76,17 +76,20 @@ type PolicySpec struct {
 
 	// matchPolicy defines how the "rules" list is used to match incoming requests.
 	// Allowed values are "Exact" or "Equivalent".
-	//
-	// - Exact: match a request only if it exactly matches a specified rule.
+	//<ul>
```

Comment:
```suggestion
	// <ul>
```

The golangci-lint run in the CI complains of the following:
https://github.com/kubewarden/kubewarden-controller/actions/runs/6482896028/job/17603316697?pr=557#step:4:13844
> commentFormatting: put a space between `//` and comment text (gocritic)


Here is the only place it happens, from what I see.